src/oscar/core/loading.py

```diff
@@ -140,27 +112,7 @@ def default_class_loader(module_label, classnames, module_prefix):
         local_module_label = installed_apps_entry + sub_module
         local_module = _import_module(local_module_label, classnames)
 
-    # Checking whether module label has corresponding move module in the MOVED_ITEMS dictionary.
-    # If it does, checking if any of the loading classes moved to another module.
-    # Finally, it they did, importing move module and showing deprecation warning as well.
-    oscar_move_item = MOVED_ITEMS.get(oscar_module_label, None)
-    if oscar_move_item:
-        oscar_move_module_label = oscar_move_item[0]
-        oscar_move_classnames = oscar_move_item[1]
-        oscar_moved_classnames = list(set(oscar_move_classnames).intersection(classnames))
-        if oscar_moved_classnames:
-            warnings.warn(
-                'Classes %s has recently moved to the new destination module - %s, '
-                'please update your imports.' % (', '.join(oscar_moved_classnames),
-                                                 oscar_move_module_label),
-                DeprecationWarning, stacklevel=2)
-            oscar_move_module = _import_module(oscar_move_module_label, classnames)
-        else:
-            oscar_move_module = None
-    else:
-        oscar_move_module = None
-
-    if oscar_module is oscar_move_module is local_module is None:
```

Comment:
I don't think we should remove the logic too - it may be useful in the future if any other classes/functions will move. We can just remove formsets from the `MOVED_ITEMS`. WDYT?

Reply:
I don't think we need all this complexity. Backward compatibility can be achieved just by aliasing classes if we ever move them. 

For example, with `apps.basket.forms.BasketLineFormSet` you would just put this into `forms.py`:

```
# Load the new class from the correct location
MovedBasketLineFormSet = get_class('basket.formsets', 'BasketLineFormSet')

# Subclass the new class into the old one, so that the old one can still be loaded.
@deprecated
class BasketLineFormSet(MovedBasketLineFormSet):
    pass
```

Then you just delete this deprecated class when the deprecation period has ended.

Reply:
This would also let you import normally from the old (deprecated) path, which you can't currently do - which means the current approach isn't fully backward compatible because any project that has subclassed the original class cannot import from the old path anyway.